### PR TITLE
fixes air_example.toml regex expression

### PR DESCRIPTION
**Pull request by jshnaidman**
I think this regex expression should be "_test\\.go". The '.' character matches any character in regex.

### CODE REVIEW

**Comment by xiantang** [comment]
could u add a test for this ? @jshnaidman

**Comment by jshnaidman** [comment]
I'm a little confused that this affects code coverage since I only updated the documentation. I added some test cases which cover the changes I made to the documentation.

**Comment by xiantang** [comment]
> I'm a little confused that this affects code coverage since I only updated the documentation. I added some test cases which cover the changes I made to the documentation.

thanks

File: air_example.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ include_dir = []
 # Exclude files.
 exclude_file = []
 # Exclude specific regular expressions.
-exclude_regex = ["_test.go"]
+exclude_regex = ["_test\\.go"]
```
Comment (xiantang) [inline air_example.toml:24]: `_test\\.go`  should be `_test\.go`

Comment (jshnaidman) [reply air_example.toml:24]: Without `` the string won't escape the \ so you need to escape it once for the "" string, and then again to escape the . So \\ I think.

Comment (jshnaidman) [reply air_example.toml:24]: Sorry I didn't realize github didn't escape, I mean't I think it should be \\\\

**Pull request merged**